examples/migration/CreateCompleteCampaignAdWordsApiOnly.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example creates serach campaign with the help of Adwords Api
+
+This code example is the first in a series of code examples that shows how to create
+a Search campaign using the AdWords API, and then migrate it to Google Ads API one
+functionality at a time. See other examples in this directory for code examples in various
+stages of migration.
+
+This code example represents the initial state, where the AdWords API is used to create a
+campaign budget, a Search campaign, ad groups, keywords and expanded text ads. None of the
+functionality has yet been migrated to the Google Ads API.
+"""
+
+
+import datetime
+import uuid
+from googleads import adwords
+import urllib.parse
```

Comment:
Can you group imports according to the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/#imports)?

```
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

You should put a blank line between each group of imports.
```
Also alphabetize them, so:

```python
import datetime
import urllib.parse
import uuid

from googleads import adwords
```